packages/fern-typescript/commons/src/dependencies/getReferenceToFernServiceUtils.ts

```diff
@@ -91,3 +91,21 @@ function addFernServiceUtilsImport({
     });
     dependencyManager.addDependency(PACKAGE_NAME, VERSION);
 }
+
+export function invokeMaybeGetter(maybeGetter: ts.Expression): ts.Expression {
```

Comment:
lol there's literally no way someone knows what this does -- my workflow was modify, run ete tests, check updated snapshots